.gitignore

```diff
@@ -218,3 +219,4 @@ numpy/core/src/_simd/_simd_data.inc
 numpy/core/src/_simd/_simd_inc.h
 # umath module
 numpy/core/src/umath/loops_unary_fp.dispatch.c
+numpy/core/src/umath/loops_arithm_fp.dispatch.c
```

Comment:
I think it is better in general to use full names like "arithmetic" rather than shortened names. Seven character names are long gone :)  This can be changed later though.

Reply:
I will consider it a policy to follow in my upcoming patches.